documentation/APIs/openapi3-core.yaml

```diff
@@ -0,0 +1,2915 @@
+            - $ref: '#/components/schemas/Gossip'
+      required:
+        - id
+        - multiAddresses
+        - relation
+        - connected
+
+    Gossip:
+      description: Information about the gossip stream with the peer.
+      properties:
+        heartbeat:
+          description: Information about the most recent heartbeat of the peer. The heartbeat is `null` if none has been received yet.
+          allOf:
+            - $ref: '#/components/schemas/Heartbeat'
+          nullable: true
+        metrics:
+          description: Metrics about the gossip stream with the peer.
+          allOf:
+            - $ref: '#/components/schemas/Metrics'
+      required:
+        - heartbeat
+        - metrics
+
+    Heartbeat:
+      properties:
+        latestCommittedSlotIndex:
+          type: integer
+          description: The most recent slot that has been committed by the node.
+        prunedSlotIndex:
+          type: integer
+          description: Tells from which starting point the node holds data.
+        latestConfirmedSlotIndex:
+          type: integer
+          description: The most recent confirmed slot known to the node.
+        connectedNeighbors:
+          type: integer
+          description: Tells how many connected peers the node has.
+        syncedNeighbors:
+          type: integer
+          description: Tells how many synced peers the node has.
+      required:
+        - latestCommittedSlotIndex
+        - prunedSlotIndex
+        - latestConfirmedSlotIndex
+        - connectedNeighbors
+        - syncedNeighbors
+
+    Metrics:
+      properties:
+        newBlocks:
+          type: integer
+          description: The number of received blocks that were new for the node.
+        knownBlocks:
+          type: integer
+          description: The number of received blocks that already were known to the node.
+        receivedBlocks:
+          type: integer
+          description: The number of received blocks from the peer.
+        receivedBlockRequests:
+          type: integer
+          description: The number of received block requests from the peer.
+        receivedHeartbeats:
+          type: integer
+          description: The number of received heartbeats from the peer.
+        sentBlocks:
+          type: integer
+          description: The number of sent blocks to the peer.
+        sentBlockRequests:
+          type: integer
+          description: The number of sent block requests to the peer.
+        sentHeartbeats:
+          type: integer
+          description: The number of sent heartbeats to the peer.
+        droppedPackets:
+          type: integer
+          description: The number of dropped packets.
+      required:
+        - newBlocks
+        - knownBlocks
+        - receivedBlocks
+        - receivedBlockRequests
+        - receivedHeartbeats
+        - sentBlocks
+        - sentBlockRequests
+        - sentHeartbeats
+        - droppedPackets
+
+    MigratedFundsEntry:
```

Comment:
Do we keep that stuff in 2.0?

Reply:
That should go away.

Reply:
Btw, can you confirm that all the following stardust types can be removed from the codebase completely please: `Receipt Milestone Option`,  `Treasury Transaction`, `Treasury Input`, `Treasury Output` and `Migrated Funds Entry`

Reply:
Yes, we don't have those in iota.go [serix-iota-core](https://github.com/iotaledger/iota.go/tree/serix-iota-core) branch.